### PR TITLE
chore(TypeScript): Change Examples.js to Examples.tsx

### PR DESCRIPTION
**Pull request by langz**
What's been done so far:
- Renamed Examples.js to Examples.tsx
- Rewrote examples to TS.

What needs to be done:
- Now that all our examples use TS it has discovered quite a few errors/issues in our d.ts files, and in our examples, which needs to be fixed/solved.

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/components/avatar/Examples.tsx [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+  <ComponentBox hideCode data-visual-test="avatar-size-default">
+    <Avatar.Group label="Persons:">
+      <Avatar>Ola Nordmann</Avatar>
+    </Avatar.Group>
+  </ComponentBox>
+)
+
+export const AvatarSizeSmall = () => (
+  <ComponentBox hideCode data-visual-test="avatar-size-small">
+    Text{' '}
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/components/avatar/Examples.tsx:24]:
```suggestion
    Text
```

I'll look into if we really need this `{' '}`. I see we have a few `{' '}` in different Examples.tsx files, but I'm not sure if we actually need them, or if they were intentional, I feel that sometimes they randomly pop up prettier and different "formatting tools".

Comment (tujoworker) [reply packages/dnb-design-system-portal/src/docs/uilib/components/avatar/Examples.tsx:24]: Yeah, we need a space there. So there are not too many other options actually.

**Review by tujoworker — APPROVED** [review]

File: packages/dnb-design-system-portal/src/docs/uilib/components/skeleton/Examples.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  if (!globalThis.IS_TEST) {
+    return <></>
+  }
+  return (
+    <>
+      <ComponentBox
+        data-visual-test="skeleton-all-horizontal"
+        scope={{ AllComponents }}
+      >
+        {`<FormRow> <Skeleton show no_animation> <AllComponents /> </Skeleton> </FormRow>`}
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/components/skeleton/Examples.tsx:107]:
```suggestion
        <FormRow> <Skeleton show no_animation> <AllComponents /> </Skeleton> </FormRow>
```

File: packages/dnb-design-system-portal/src/docs/uilib/components/skeleton/Examples.tsx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+      </ComponentBox>
+      <ComponentBox
+        data-visual-test="skeleton-all-vertical"
+        scope={{ AllComponents }}
+      >
+        {`<FormRow vertical="true"> <Skeleton show no_animation> <AllComponents /> </Skeleton> </FormRow>`}
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/components/skeleton/Examples.tsx:113]:
```suggestion
        <FormRow vertical="true"> <Skeleton show no_animation> <AllComponents /> </Skeleton> </FormRow>
```

File: packages/dnb-design-system-portal/src/docs/uilib/components/avatar/Examples.tsx (hunk 1 above)
Comment (langz) [reply packages/dnb-design-system-portal/src/docs/uilib/components/avatar/Examples.tsx:24]: Okok, if we need a space, then it's of course correct 👍 
I just assumed the space was not needed in `AvatarSizeSmall`, as we don't have it in `AvatarSizeMedium`, `AvatarSizeLarge`, and `AvatarSizeXLarge`.

Comment (langz) [reply packages/dnb-design-system-portal/src/docs/uilib/components/avatar/Examples.tsx:24]: Never mind, it's all good.
I think this is good yes ✅

File: packages/dnb-design-system-portal/src/docs/uilib/components/skeleton/Examples.tsx (hunk 2 above)
Comment (langz) [reply packages/dnb-design-system-portal/src/docs/uilib/components/skeleton/Examples.tsx]: https://github.com/dnbexperience/eufemia/pull/2007/commits/77ada66b20682f68379836c957cc5e2a0f7d8464

File: packages/dnb-design-system-portal/src/docs/uilib/components/skeleton/Examples.tsx (hunk 3 above)
Comment (langz) [reply packages/dnb-design-system-portal/src/docs/uilib/components/skeleton/Examples.tsx]: https://github.com/dnbexperience/eufemia/pull/2007/commits/77ada66b20682f68379836c957cc5e2a0f7d8464

**Pull request merged**